.github/workflows/schema-linter.yml

```
@@ -45,10 +45,10 @@ jobs:
           composer run install-test-env
 
       - name: Generate the Static Schema
-        # 💡 Neutralize for now
+        # TODO: fix remaining exceptions
         run: |
           cd /tmp/wordpress/
           # Output: /tmp/schema.graphql
           wp graphql generate-static-schema
           cd /tmp/wordpress/wp-content/plugins/wp-graphql/
-          graphql-schema-linter /tmp/schema.graphql || true
+          graphql-schema-linter --ignore '{"relay-page-info-spec":["PageInfo"],"defined-types-are-used":["MenuItemsWhereArgs","PostObjectUnion","TermObjectUnion","TimezoneEnum"]}' /tmp/schema.graphql
```

```suggestion
          graphql-schema-linter --except=relay-connection-types-spec --ignore '{"relay-page-info-spec":["PageInfo"],"defined-types-are-used":["MenuItemsWhereArgs","PostObjectUnion","TermObjectUnion","TimezoneEnum"]}' /tmp/schema.graphql
```

If you exclude `relay-connection-types-spec` the rule globally - as discussed in the top comment - we could see the remaining problems.

---

Maybe only two!!

---

Here's how I've been running the linter locally and testing the rules.

```
# Install node graphql linter
npm install graphql-schema-linter

# Start the WP app
# Verify the WP is up by visiting http://localhost:8091/
composer run-app

# Generate the schema in the WP, copy the file to an accessible location, then run the node linter command
docker-compose exec -- app /usr/local/bin/wp --allow-root graphql generate-static-schema
docker-compose exec -- app cp /tmp/schema.graphql /var/log/apache2/
node_modules/.bin/graphql-schema-linter .log/app/schema.graphql
```

---

Okay. Just add `--except` and `--ignore`.